models/public/pspnet-pytorch/model.yml

```diff
@@ -0,0 +1,160 @@
+    size: 1261
+    sha256: 8ac435c6388d9896dacde51e0f24d3d3c27f4972dad88f1eba9327341b689c25
+    source: https://raw.githubusercontent.com/open-mmlab/mmsegmentation/993be2523b908d1e0cb45bc68e92d210b5b2cda1/configs/_base_/models/pspnet_r50-d8.py
+  - name: models/configs/_base_/datasets/pascal_voc12_aug.py
+    size: 261
+    sha256: b61d01edccac1a2622944761a50772a54b4848cd9e4817c25551f2cccdd05a08
+    source: https://raw.githubusercontent.com/open-mmlab/mmsegmentation/993be2523b908d1e0cb45bc68e92d210b5b2cda1/configs/_base_/datasets/pascal_voc12_aug.py
+  - name: models/configs/_base_/datasets/pascal_voc12.py
+    size: 1930
+    sha256: b9fac8609d3c8d8a3536803b53a1222ba2d6f02af6b1638b7e78af4ab233805b
+    source: https://raw.githubusercontent.com/open-mmlab/mmsegmentation/993be2523b908d1e0cb45bc68e92d210b5b2cda1/configs/_base_/datasets/pascal_voc12.py
+  - name: models/configs/_base_/default_runtime.py
+    size: 321
+    sha256: 27e0d1e40cf3956e2af35b8de32436eb78f91715570ba435144b29b515b1b5a1
+    source: https://raw.githubusercontent.com/open-mmlab/mmsegmentation/993be2523b908d1e0cb45bc68e92d210b5b2cda1/configs/_base_/default_runtime.py
+  - name: models/mmsegmentation-0.11.0-py3-none-any.whl
+    size: 147357
+    sha256: 5b28a1b1c84fef8a28ac6852ef72dfab6f56dc13c33b4b24ebb358f4ea003a4a
+    source: https://files.pythonhosted.org/packages/b4/f9/4722a9dfc9bf2d6d6f5cabb90cd61d549ae316ccbe226fc6e245b586d5f9/mmsegmentation-0.11.0-py3-none-any.whl
+  - name: models/addict-2.4.0-py3-none-any.whl
+    size: 3832
+    sha256: 249bb56bbfd3cdc2a004ea0ff4c2b6ddc84d53bc2194761636eb314d5cfa5dfc
+    source: https://files.pythonhosted.org/packages/6a/00/b08f23b7d7e1e14ce01419a467b583edbb93c6cdb8654e54a9cc579cd61f/addict-2.4.0-py3-none-any.whl
+  - name: models/mmcv-1.2.0.tar.gz
+    size: 241987
+    sha256: 2e34bcacd977f101f98447be08e565c19aedeedadc155e97f8f75efe356b7ca9
+    source: https://files.pythonhosted.org/packages/d4/a0/01de45397c6222a29cf088bf54d4fb5038a4a524340eef0f923335a23116/mmcv-1.2.0.tar.gz
+postprocessing:
+  - $type: unpack_archive
+    format: gztar
+    file: models/mmcv-1.2.0.tar.gz
+  - $type: unpack_archive
+    format: zip
+    file: models/addict-2.4.0-py3-none-any.whl
+  - $type: unpack_archive
+    format: zip
+    file: models/mmsegmentation-0.11.0-py3-none-any.whl
+
+  - $type: regex_replace
+    pattern: '''../_base_/schedules/schedule_20k.py'''
+    replacement: '# \g<0>'
+    file: models/configs/pspnet/pspnet_r50-d8_512x512_20k_voc12aug.py
+  - $type: regex_replace
+    pattern: 'isinstance\(var, list\)'
+    replacement: 'isinstance(var, (list, torch.Tensor))'
+    file: models/mmseg/models/segmentors/base.py
+  - $type: regex_replace
+    pattern: 'num_augs = len\(imgs\)'
+    replacement: 'num_augs = 1'
+    file: models/mmseg/models/segmentors/base.py
+  - $type: regex_replace
+    pattern: 'imgs\[0\]'
+    replacement: 'imgs'
+    file: models/mmseg/models/segmentors/base.py
```

Comment:
Could this, perhaps, be replaced with a wrapper model class in `model.py` that takes an image, wraps it in a list and passes it to the underlying model? It would be clearer than this.

Reply:
Done